package.json

```diff
@@ -11,28 +11,34 @@
   "private": true,
   "scripts": {
     "build": "gulp",
-    "watch": "gulp watch"
+    "watch": "gulp watch",
+    "check": "jest"
   },
   "dependencies": {
     "clipboard": "^1.5.12",
-    "react": "^15.3.2",
-    "react-dom": "^15.3.2"
+    "react": "^15.3.1",
```

Comment:
Seems like this file wasn't merged properly and doesn't have the latest changes.